docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,250 @@
+# Creating a tool with an API hit
+In this guide, we will use the [Alteryx Python SDK](https://pypi.org/project/ayx-python-sdk/) and [Alteryx Plugin CLI](https://pypi.org/project/ayx-plugin-cli/) to create a tool that pulls information from an API and find the mean, min, and max of the data.
+
+---
+## Workspace Setup
+---
+### 1. Creating a workspace
+The very first step to creating a plugin is to make a plugin workspace. We initialize a plugin workspace in an empty directory with the `sdk-workspace-init` command. Run the command and fill out the prompts, which will then start the workspace initialization process.
```

Comment:
TW review - update to:

The first step of plugin creation is to make a plugin workspace. Use the `sdk-workspace-init` command to initialize a plugin workspace in an empty directory. Run the command and reply to the prompts, which will then start the workspace initialization process.

Reply:
I think it might be confusing to say "Use the sdk-workspace-init command to initialize a plugin workspace in an empty directory." Would it be better to make it really explicit that the user needs to: 
1. create a new directory (which is empty) and then 
2. run the command inside of it

Reply:
@ayx-orion I like your suggestion. Does something like this work?

1. Create a new, empty directory.
2. Run the sdk-workspace-init command in the directory.

Reply:
yeap i like that
how's this sound:

We initialize a plugin workspace by first creating a new, empty directory. Then, run the `sdk-workspace-init` command inside that directory.

Reply:
I think we're getting close! I made a small update:

To initialize the plugin workspace, we first create a new, empty directory. Then, we run the `sdk-workspace-init` command inside that directory.

Reply:
added the last bit about filling out prompts

To initialize the plugin workspace, we first create a new, empty directory. Then, we run the sdk-workspace-init command inside that directory. Fill out the prompts, which then kicks off the workspace initialization process.